scripts/win-ci-logs-collector.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "$#" -ne 1 ]; then
+    echo "Unexpected number of parameters."
+    echo "Usage: win-ci-logs-collector.sh masterIp"
+    exit 1
+fi
+
+MASTER_IP=${1}
+USER="azureuser"
+
+# Check for kubectl command
+if ! command -v kubectl > /dev/null; then
+  echo "kubectl not found"
+  exit 1
+fi
+
+# Check for ssh key in path
+if [ -z "$SSH_KEY" ]
+then
+    echo "ssh key not found. Defaulting to .ssh/id_rsa."
+    SSH_KEY="${HOME}/.ssh/id_rsa"
+fi
+
+if [ ! -f "$SSH_KEY" ]
+then
+    echo "ssh key file ${SSH_KEY} does not exist. Exiting."
+    exit 1
+fi
+
+# Collect master provisioning logs
+master_logs_output="${ARTIFACTS}/master_provisioning"
+readonly master_provisioning_logs=(
+    "/tmp/mastier_extension.log"
```

Comment:
master_extension.log (unless the extension is actually logging to mastier_extension.log)